zuul.d/jobs.yaml

```diff
@@ -332,3 +332,14 @@
     run: playbooks/cf-openstack-validator-acceptance-test/run.yaml
     vars:
       os_branch: 'stable/newton'
+
+- job:
+    name: bosh-acceptance-tests
```

Comment:
nits: job name and directory name should be s/tests/test/ , keep consistence.

Reply:
done